packit_service/worker/github_handlers.py

```diff
@@ -78,6 +78,27 @@ class AbstractGithubJobHandler(JobHandler):
     pass
 
 
+def _get_package_config_from_repo(
+    project: GitProject, reference: str, pr_id: int = None
+):
+    try:
+        package_config: PackageConfig = get_package_config_from_repo(project, reference)
+        if not package_config:
+            raise PackitConfigException(
+                f"No config file found in {project.full_repo_name}"
+            )
+    except PackitConfigException as ex:
+        if pr_id:
+            project.pr_comment(pr_id, ex)
+        else:
+            issues = project.get_issue_list()
```

Comment:
this will be pretty beefy on projects with thousands of issues

Reply:
If GitHub supports filtering, this will be a nice improvement to OGR..;)

Reply:
https://github.com/packit-service/ogr/issues/308